.circleci/config.yml

```diff
@@ -251,15 +254,21 @@ jobs:
             just simulate-council # simulate again to make sure the json is still valid
 
   just_simulate_ink_respected_game_type:
+    circleci_ip_ranges: true
     docker:
       - image: <<pipeline.parameters.default_docker_image>>
+    environment:
+      ETH_RPC_URL: << pipeline.parameters.l1_mainnet_rpc_url >>
     steps:
       - simulate:
           task: "/eth/ink-002-set-respected-game-type"
 
   just_simulate_zora_002_fp_upgrade:
+    circleci_ip_ranges: true
     docker:
       - image: <<pipeline.parameters.default_docker_image>>
+    environment:
+      ETH_RPC_URL: << pipeline.parameters.l1_sepolia_rpc_url >>
```

Comment:
Do we need this parameter, or can we infer this from the task path in the `simulate` and `simulate-nested` commands and set it there? Or we can add `network` as a another parameter that way we just need to specify "mainnet" or "sepolia" instead of an env var